### PR TITLE
Sync midstream changes

### PR DESCRIPTION
**Pull request by chmeliik**
From #6

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,4 +22,16 @@ LABEL org.opencontainers.image.description="CLI tool and library for generating
 LABEL org.opencontainers.image.vendor="Red Hat, Inc."
 LABEL org.opencontainers.image.licenses="Apache-2.0"
 
+# required per https://github.com/release-engineering/rhtap-ec-policy/blob/main/data/rule_data.yml
+LABEL com.redhat.component="TODO: The Bugzilla component name where bugs against this container should be reported by users."
```
Comment (chmeliik) [inline Dockerfile:26]: How to get a bugzilla component?

File: Dockerfile [hunk 2]
```diff
@@ -22,4 +22,16 @@ LABEL org.opencontainers.image.description="CLI tool and library for generating
...
+LABEL version="0.89.0"
+# TODO: how do we make sure to bump this
+LABEL release="1"
+LABEL name="syft"
+LABEL description="CLI tool and library for generating a Software Bill of Materials from container images and filesystems"
+LABEL io.k8s.description="CLI tool and library for generating a Software Bill of Materials from container images and filesystems"
+LABEL vendor="Red Hat, Inc."
+LABEL url="TODO: A URL where the user can find more information about the image."
```
Comment (chmeliik) [inline Dockerfile:34]: AFAIK this is the catalog page for most RH containers. Can it just be a link to this repo?

File: Dockerfile [hunk 3]
```diff
@@ -22,4 +22,16 @@ LABEL org.opencontainers.image.description="CLI tool and library for generating
...
+LABEL distribution-scope="TODO: Scope of intended distribution of the image. (private/authoritative-source-only/restricted/public)."
```
Comment (chmeliik) [inline Dockerfile:35]: Which one for tech preview?

**Comment by chmeliik** [comment]
PTAL

The 1 failing EC check is a dependency with a CVE, which I'm not planning to address. The CVE is not present in Syft 0.100.0 anymore, but we can't build that one hermetically yet (need https://github.com/containerbuildsystem/cachi2/pull/427).

In other words, I'll be ignoring that one and merging things anyway to make some progress with the release machinery

**Comment by chmeliik** [comment]
You can also review the related #6 instead - I'll take your approval on either of these as approval for both :slightly_smiling_face: 

As for why we need two PRs, see the README (maybe with the updates from #5)

**Review by tkdchen — APPROVED** [review]

**Pull request merged**